sound/hda/intel-dsp-config.c

```diff
@@ -17,6 +17,11 @@ static int dsp_driver;
 module_param(dsp_driver, int, 0444);
 MODULE_PARM_DESC(dsp_driver, "Force the DSP driver for Intel DSP (0=auto, 1=legacy, 2=SST, 3=SOF)");
 
+static int acpi_dsp_driver;
+
+module_param(acpi_dsp_driver, int, 0444);
+MODULE_PARM_DESC(dsp_driver, "Force the DSP driver for Intel ACPI DSP (0=auto, 2=SST, 3=SOF)");
```

Comment:
Copy-and-paste typo here: `MODULE_PARM_DESC(acpi_dsp_driver,`

Reply:
Thanks for catching this @perexg. I actually want to use a single parameter, and ignore the 'legacy' case for ACPI. Will remove this.

Reply:
fixed